From dd54b7a97d6ca0b4502298703659266058fd464b Mon Sep 17 00:00:00 2001
From: Dan Prince <dprince@redhat.com>
Date: Mon, 6 Oct 2014 17:54:54 -0400
Subject: [PATCH 2/2] Revert "Switch over to oslosphinx"

From: Eric Harney <eharney@redhat.com>
Date: Thu, 27 Mar 2014 14:24:57 -0400
Subject: [PATCH] Revert "Switch over to oslosphinx"

NOTE: no need to patch test-requirements.txt since we
remove that file in the spec anyways.
---
 doc/source/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index d2ea1ce..c6f666a 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -32,7 +32,7 @@ extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.pngmath',
               'sphinx.ext.ifconfig',
               'sphinx.ext.graphviz',
-              'oslosphinx',
+              'oslo.sphinx',
               ]
 
 # autodoc generation is a bit aggressive and a nuisance
-- 
1.9.3

